app/src/main/res/layout/fragment_feed.xml

```diff
@@ -62,15 +63,15 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/rvPosts"
-        android:layout_width="402dp"
-        android:layout_height="668dp"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
```

Comment:
just double-checking - is this intentional? do you know why these are supposed to be 0dp?

Reply:
Yes it was. A constraint not to have any space in the height and width.

Reply:
Yes it was. A constraint not to have any space in the height and width.